src/main/target/CLRACINGF4/target.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
You don't need `src/main/target/CLRACINGF4/CLRACINGF4.mk`. This is only required for variants.